packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -43,39 +63,39 @@
 }
 
 :host(:active) .anchor {
-  @apply text-color-1;
+  color: var(--calcite-navigation-logo-text-color-active, var(--calcite-color-text-1));
 }
 
 :host([active]) .anchor {
-  @apply text-color-1;
-  border-color: var(--calcite-color-brand);
+  color: var(--calcite-navigation-logo-active-text-color, var(--calcite-color-text-1));
```

Comment:
if you are going to use stateful tokens and not #1 or #2 listed above, this needs to be `--calcite-navigation-logo-text-color-active`